From ca6c67b6c307988c55f1179e4bcb208d7e1b9ee7 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Fri, 10 Mar 2017 19:52:36 +0800
Subject: [PATCH 1576/5242] MLK-14471-1 arm: dts: restore touch pin when exit
 VLLS mode

commit  2b72618caacd2017e5528f0cc9f92c0f3daad262 from
https://source.codeaurora.org/external/imx/linux-imx.git

restore touch pin when exit VLLS mode

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
(cherry-pick from 807fdc794af9b6ec15673d5e70cb03065b80f9fd)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index fb71dcc..5f2a332 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -367,14 +367,14 @@
 &lpi2c7 {
 	#address-cells = <1>;
 	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_lpi2c7>;
+	pinctrl-names = "default",  "sleep";
+	pinctrl-0 = <&pinctrl_lpi2c7 &pinctrl_focaltech>;
+	pinctrl-1 = <&pinctrl_lpi2c7 &pinctrl_focaltech>;
 	status = "okay";
 
 	focaltech@38 {
 		compatible = "focaltech,fts";
 		reg = <0x38>;
-		pinctrl-0 = <&pinctrl_focaltech>;
 		interrupt-parent = <&gpio3>;
 		interrupts = <0 0x02>;
 		focaltech,reset-gpio = <&gpio3 1 0x01>;
-- 
1.7.9.5

